myhpi/templates/base.html

```diff
@@ -52,18 +52,23 @@
                         role="button">
                         {% bs_icon 'search' size='1.4em' %}
                     </a>
-                    <a id="nav-user-toggle" class="nav-link" href="#nav-item-container-user"
-                        onclick="toggleUserNavbar(event)"
-                        role="button"
-                        aria-expanded="false" aria-controls="nav-item-container-user">
-                        {% bs_icon 'person-circle' size='1.4em' %}
-                    </a>
-                    <a id="nav-mobile-toggle" class="nav-link d-xl-none" href="#nav-item-container-root"
-                        onclick="toggleMobileNavbar(event)"
-                        role="button"
-                        aria-expanded="false" aria-controls="nav-item-container-root">
-                        {% bs_icon 'list' size='1.4em' %}
-                    </a>
+                    {% if user.is_authenticated %}
+                        <a id="nav-user-toggle" class="nav-link" href="#nav-item-container-user"
+                            onclick="toggleUserNavbar(event)"
+                            role="button"
+                            aria-expanded="false" aria-controls="nav-item-container-user">
+                            {{ user.first_name }} {{ user.last_name }}
+                            {% bs_icon 'person-circle' size='1.4em' %}
+                        </a>
+                        <a id="nav-mobile-toggle" class="nav-link d-xl-none" href="#nav-item-container-root"
+                            onclick="toggleMobileNavbar(event)"
+                            role="button"
+                            aria-expanded="false" aria-controls="nav-item-container-root">
+                            {% bs_icon 'list' size='1.4em' %}
+                        </a>
+                    {% else %}
+                        <a id="nav-user-toggle" class="nav-link" class="dropdown-item" href="{% url 'oidc_authentication_init' %}{% if 'next' in request.GET %}?next={{ request.GET.next }}{% endif %}">Login</a>
```

Comment:
I'd prefer using "Anmelden" and "Abmelden" in German instead of "Login"/"Logout", but no hard opinion there

We could also use an icon plus tooltip "Login" instead of the text "Login" (imo having text next to the search icon looks a bit off). This boostrap icon might fit: [box-arrow-in-right](https://icons.getbootstrap.com/icons/box-arrow-in-right/).  
If we want to stick to the Login as text, we should make sure that there is no padding on the right, so the "Login" text is aligned with the website content below

Reply:
Agreed. This will be implemented with the localisation (#224). 

I will add the logo to improve the design.